src/lib.rs

```diff
@@ -20,11 +20,10 @@
 
 //! A crate used for calculating offsets of struct members and their spans.
 //!
-//! Some of the funcationality of the crate makes no sense when used along with structs that
+//! Some of the functionality of the crate makes no sense when used along with structs that
 //! are not `#[repr(C, packed)]`, but it is up to the user to make sure that they are.
```

Comment:
I am actually quite puzzled by this remark about `(C, packed)`... but that's orthogonal to this PR.

@Gilnaa what is that remark about?

Reply:
That remark was about me being wrong.